vipps-checkout-how-it-works-shipping.md

```diff
@@ -17,27 +19,32 @@ A provider is chosen by setting the `brand` property to one of the allowed strin
 
 ![Shipping provider logo example](resources/shipping_logo-example.png)
 
-If none of the providers fit your use-case, e.g. indicating in-store-pickup, setting `brand: "OTHER"` will set a generic icon.
+If none of the providers fit your use case (e.g., indicating in-store-pickup), you can set a generic icon by setting `brand: "OTHER"`.
```

Comment:
You have to set brand, so "OTHER" must be used for generic shipping

Reply:
Something like this? 

`If none of the providers fit your use case (e.g., indicating in-store-pickup), set the brand to "OTHER" for generic shipping. This will set a generic icon.`

? or just almost the same as the original:
`If none of the providers fit your use case (e.g., indicating in-store-pickup), setting `brand: "OTHER"` will set a generic icon.`

Reply:
I like 
"If none of the providers fit your use case (e.g., indicating in-store-pickup), set the brand to "OTHER" for generic shipping. This will set a generic icon."